lib/sites.ts

```diff
@@ -30,7 +30,8 @@ export function useSites() {
     `${process.env.NEXT_PUBLIC_API_BASE_URL_GET}/sites`
   );
 
-  return { sites: sites?.site_list ?? [], error, isLoading };
+  // TODO: Paginate this globally somehow. Right now there is too much site data being fetched on the aggregate dashboard
+  return { sites: sites?.site_list?.slice(0, 5) ?? [], error, isLoading };
```

Comment:
this probably won't be too much of an issue with individual homeowners  right? but yeah I think we should make this into an issue so that this problem is easier to find.